runtime/lib.h

```diff
@@ -308,6 +308,17 @@ struct ExplicitValueOrControlFlow {
     _jakt_value.release_value(); \
 })
 
+#define JAKT_RESOLVE_EXPLICIT_VALUE_OR_CONTROL_FLOW_AT_LOOP_NESTED_MATCH(x) ({ \
+    auto&& _jakt_value = x; \
+    if (_jakt_value.is_return()) \
+      return _jakt_value.release_return(); \
+    else if (_jakt_value.is_loop_break()) \
+      return JaktInternal::LoopBreak{}; \
+    else if (_jakt_value.is_loop_continue()) \
+      return JaktInternal::LoopContinue{}; \
+    _jakt_value.release_value(); \
+})
+
```

Comment:
Can you please run `clang-format` on this?

Reply:
Yeah, thanks for pointing out.

Reply:
Also you couldn't notice that because I added it to solve the bug I noticed later :)